src/node/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitOverride": true,
     "noImplicitReturns": true,
-    "noPropertyAccessFromIndexSignature": true,
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
Note to reviewer:

I'm not quite sure why we enabled this in the first place. A object with a type of `Record<unknown, unknown>` has to access properties using `obj['my_field']` rather than `obj.my_field`. Without this change, we create a lot of unnecessary strings.